tests/test_scrapy.py

```diff
@@ -60,14 +60,14 @@ def test_request_response_converters():
 
     frontier_response = rsc.to_frontier(response)
     assert frontier_response.body == RESPONSE_BODY
-    assert frontier_response.meta[b'scrapy_meta'][b'test_param'] == b'test_value'
+    assert frontier_response.meta[b'scrapy_meta']['test_param'] == b'test_value'
```

Comment:
Although in this case we can go either way, since frontera doesn't currently acess `scrapy_meta`, it might be better to follow the convention and have everything possible as bytes. So I would suggest a conversion to `bytes` for both keys and values.